### PR TITLE
Adds *yip and *yipyip

### PR DESCRIPTION
**Pull request by CydiaButt13**
yip yip!

## About The Pull Request

It adds *yip and *yipyip, because I want the kobolds to make NOISE.

## Proof Of Testing

https://github.com/user-attachments/assets/d7baa626-656e-4a46-b300-d2314fdc8796

https://github.com/user-attachments/assets/acff25e6-5061-486e-86b8-d385b92debee

## Changelog

:cl:
add: *yip
add: *yipyip
/:cl:

### CODE REVIEW

**Comment by nevimer** [comment]
yip

**Comment by nevimer** [comment]
was this sound sourced from kobold generator?

**Comment by CydiaButt13** [comment]
> was this sound sourced from kobold generator?

I got it from a soundboard, looking up "kobold yip noise". It might've been from that, not sure

**Comment by nevimer** [comment]
suggestion for you: split *yip into a single yip, and *yipyip into the double yip, the current file

**Comment by CydiaButt13** [comment]
Better idea, I can make it modular with different noises iirc, hold on a second

**Comment by CydiaButt13** [comment]
OKAY

https://github.com/user-attachments/assets/dfd951cb-b0d3-4f99-8e3a-67488fd199be

File: modular_skyrat/modules/emotes/code/emotes.dm
Comment (nevimer) [inline modular_skyrat/modules/emotes/code/emotes.dm:591]: These share the same message, but otherwise I think it's good to go

**Review by nevimer — APPROVED** [review]
Might need to cite the source of the sounds

**Comment by YakumoChen** [comment]
Sounds should be source-cited, to ensure you are not treading on copyright

**Comment by CydiaButt13** [comment]
Do you want me to just cite it in the PR or in the emote itself? like a //

**Comment by YakumoChen** [comment]
> Do you want me to just cite it in the PR or in the emote itself? like a //

Depends on the sound's license, but I see a lot of sounds just use a license file like #2251

**Comment by CydiaButt13** [comment]
Currently waiting on kobold generators response through dm, I don't see their creative commons anywher

**Comment by IgiariValkyr** [comment]
> was this sound sourced from kobold generator? https://introdile.itch.io/kobold-generator

Proper source would be DnD: Online, from an old dungeon that hasn't been active in years.

**Comment by YakumoChen** [comment]
https://wyx.gay/yipbot

https://cohost.org/alizard/post/1291278-any-kobolds-have-a-y

This guy's yip sound might be up for grabs, if you want an easier time of something more easy to get permission for

**Comment by CydiaButt13** [comment]
Got a reply from the creator, said they had to ask their friend who recorded the yips, but they seemed alright with it, will report back o7

**Comment by nevimer** [comment]
ah, one thing before I merge it, let's move it out of modular skyrat into modular zubbers?

**Comment by YakumoChen** [comment]
> > if its ripped from DnD then I doubt WotC would appreciate us open sourcing it
> > Just putting the source in your PR is not _permission_ to use it, keep in mind.
> > When you are using a license file you should be including the actual license used for the sound, not just the source.
> > It would be like ripping a pokemon sprite and linking to gamefreak's website, for a game with a license that explicitly makes the asset freely available for everyone (which is what this codebase does)
> 
> i dont think its a huge deal given skyrat loaded up their repo with tons of valve assets, not to mention all sorts of other things

don't agree, just because skyrat maintainers are bad at keeping good copyright practices does not mean that poor behavior should continue

**Comment by CydiaButt13** [comment]
Okay well I got permission, it's alright
image

**Pull request merged**
